Library/Homebrew/formula.rb

```diff
@@ -3026,6 +3032,8 @@ def inherited(child)
         @skip_clean_paths = Set.new
         @link_overwrite_paths = Set.new
         @loaded_from_api = false
+        @supported_phases = [:build, :test, :postinstall].freeze
```

Comment:
```suggestion
        @supported_network_access_phases = [:build, :test, :postinstall].freeze
```

or something. Also feels like this might be nicer as a constant?